recipes/protopuf/all/test_package/CMakeUserPresets.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file should not be added to version control. We must need to add this to the `.gitignore`.

Reply:
I've submitted a PR to add this to the `.gitignore`: #13348

Reply:
@jwillikers 
Thanks!
I removed CMakeUserPresets.json from this PR.